vm_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procfs
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestVm(t *testing.T) {
+	fs, err := NewFS(procTestFixtures)
+	if err != nil {
+		t.Fatal(err)
+	}
+	stats, err := fs.Vm()
+	if err != nil {
+		t.Fatal(err)
+	}
+	var (
+		adminReserveKbytes        int64 = 8192
```

Comment:
Remove this var block and move all of these non-zero values into the struct declaration below to cut a lot of redundancy. You can omit any zero values from the struct declaration as they are inferred as the zero value of int64 (0) anyway.

Reply:
right, also a copy&paste artefact